core/src/com/unciv/logic/trade/TradeLogic.kt

```diff
@@ -83,6 +83,17 @@ class TradeLogic(val ourCivilization: Civilization, val otherCivilization: Civil
             }
         }
 
+        if (!civInfo.isCityState && !otherCivilization.isCityState) {
+            val thirdCivsAtWarWeKnow = otherCivilization.getKnownCivs()
+                .filter { it.civName != otherCivilization.civName && !it.isDefeated() && it.isAtWarWith(civInfo) }
```

Comment:
otherCivilization.getKnownCivs does not return itself
Maybe you mean "isn't me"?

```suggestion
                .filter { it.civName != civInfo.civName && !it.isDefeated() && it.isAtWarWith(civInfo) }
```